--- make/autoconf/build-aux/config.guess	2021-08-05 15:12:27.000000000 -0700
+++ make/autoconf/build-aux/config.guess	2021-09-20 21:49:23.669618540 -0700
@@ -56,15 +56,6 @@
   esac
 fi
 
-# Test and fix wsl
-echo $OUT | grep unknown-linux-gnu > /dev/null 2> /dev/null
-if test $? = 0; then
-  uname -r | grep -i microsoft > /dev/null 2> /dev/null
-  if test $? = 0; then
-    OUT=`echo $OUT | sed -e 's/unknown-linux-gnu/pc-wsl/'`
-  fi
-fi
-
 # Test and fix architecture string on AIX
 # On AIX 'config.guess' returns 'powerpc' as architecture but 'powerpc' is
 # implicitely handled as 32-bit architecture in 'platform.m4' so we check
